src/main/resources/templates/layouts/main.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
```

Comment:
I missed that. It's incorrect.
```suggestion
<html lang="en">
```